### PR TITLE
Move fast forward to backplane-2.1

### PR DESCRIPTION
**Pull request by jnpacker**
Signed-off-by: Joshua Packer <jpacker@redhat.com>

### CODE REVIEW

**Review by rokej — APPROVED** [review]

**Comment by hongkailiu** [comment]
The error there is about the prow-job files sharding. This is different from you have sharded on the ci-operator config files.

```
time="2022-04-29T17:27:57Z" level=error msg="Validation failed" error="ci-operator/jobs/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-backplane-2.1-postsubmits.yaml.config_updater.maps: Invalid value: \"ci-operator/jobs/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-backplane-2.1-postsubmits.yaml\": Config file does not belong to any auto-updating config."
```

**Comment by schmidtd** [comment]
@jnpacker Did/can you run `make update` at the top of your `release` repo?  If so, does it produce changes that need to be pushed in this PR?

**Comment by jnpacker** [comment]
@schmidtd @[hongkailiu] I updated to the latest commit from `master`, I'm still seeing this error:
```
 time="2022-05-11T15:03:32Z" level=error msg="Validation failed" error="ci-operator/jobs/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-backplane-2.1-postsubmits.yaml.config_updater.maps: Invalid value: \"ci-operator/jobs/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-backplane-2.1-postsubmits.yaml\": Config file does not belong to any auto-updating config." 
```
So is this related to my other PR to add backplane-2.1 to the plugins?

**Comment by zhujian7** [comment]
> I updated to the latest commit from `master`, I'm still seeing this error:

I guess maybe we should also update the job-config-misc [here](https://github.com/openshift/release/pull/28137/files#diff-a9035bbaee5af66130c81cfd0b97df589a7d2e22e87ad9d3800851407967c07bR1600)

**Comment by jnpacker** [comment]
That was this Pull request: https://github.com/openshift/release/pull/28140/files

That @schmidtd suggested needed to be merged first, but it requires an OpenShift CI CD admin to merge it.?? (is that right)?

File: ci-operator/config/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-backplane-2.1.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
+base_images:
+  stolostron_builder_go1.17-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
```
Comment (zhujian7) [inline ci-operator/config/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-backplane-2.1.yaml:5]: s/go1.17/go1.18/g

File: ci-operator/config/stolostron/hypershift-deployment-controller/stolostron-hypershift-deployment-controller-backplane-2.1.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+base_images:
+  stolostron_builder_go1.17-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
```
Comment (zhujian7) [inline ci-operator/config/stolostron/hypershift-deployment-controller/stolostron-hypershift-deployment-controller-backplane-2.1.yaml:5]: ditto

**Comment by jnpacker** [comment]
@zhujian7 I suspect e2e fails because its not our actual build?

**Comment by zhujian7** [comment]
hypershift-addon-operator/main/e2e failed, the reason is preparing the ocp cluster timeout.
```
*[[20](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/28137/rehearse-28137-pull-ci-stolostron-hypershift-addon-operator-backplane-2.1-e2e/1529164805842145280#1:build-log.txt%3A20)22-05-24T18:29:09Z]NoClusters: No clusters in pool are ready to be claimed 
INFO[2022-05-24T19:29:09Z] Releasing cluster claims for test e2e        
INFO[2022-05-24T19:29:09Z] Ran for 1h11m1s
```
Now we set the timeout 1h, https://github.com/openshift/release/blob/master/ci-operator/config/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-main.yaml#L54 ,maybe we need to change it to 1h30m0s

**Comment by ray-harris** [comment]
@jnpacker Rehearsal jobs for our repos are not expected to pass. They're not required and can generally be ignored.

**Pull request merged**